README.md

```diff
@@ -290,6 +290,21 @@ require("grapple").setup({
         -- Custom: adds padding around window title
         title_padding = " ",
     },
+
+    --Override default floating window mappings
+    win_mappings = {
+        toggle_hidden = "g.",
```

Comment:
Some action name change suggestions:
- `unload_scope` -> `unload` and put in the `loaded_mappings` table
- `reset_scope` -> `reset` and put in the `loaded_mappings` table
- `change_scope` -> `change` and put in the `scope_mappings` table
- `select_horizontal` -> `select_split` and put in the `tag_mappings` table
- `select_vertical` -> `select_vsplit` and put in the `tag_mappings` table

Reply:
thanks for the insights, I'll change it

Reply:
did you list all the renames here? Or is something left